.travis.yml

```
@@ -9,7 +9,7 @@ addons:
 install:
   - luarocks --local install luafilesystem
   - luarocks --local install luacheck
-  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc --no-check-certificate https://builds.cuberite.org/job/Cuberite%20Windows%20x64%20Master/lastSuccessfulBuild/artifact/Server/.luacheckrc
+  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc https://ci.appveyor.com/api/projects/cuberite/cuberite/artifacts/.luacheckrc
```

```suggestion
  - wget -O $TRAVIS_BUILD_DIR/.luacheckrc https://ci.appveyor.com/api/projects/cuberite/cuberite/artifacts/.luacheckrc?job=Windows-x64&pr=false&branch=master
```
We don't want to pull the file from PRs and branches other than master.